### ISSUE TITLE
GitLab integration 

### ISSUE THREAD

**Issue by pedrommone** [dependenci/roadmap#15]
Would be wonderful if we can have more than GitHub!

**Comment by m1guelpf** [comment]
@pedrommone In progress!

**Comment by m1guelpf** [comment]
@pedrommone Added to beta

**Comment by pedrommone** [comment]
Hey, sorry for such delay. I'll take a look a on that right now :)

**Comment by pedrommone** [comment]
@m1guelpf hello! Thanks for building this wonderful tool! :)

There are a few things that I think will be nice:
- Let me choose someone to assign the merge requests;
- Respect the `semver` pattern (http://imgur.com/a/qCmNS).

Also, we would love it to auto-upgrade.

**Comment by iwex** [comment]
@pedrommone you mean, like to do some checkboxes with what to update (major, minor, patch)?

**Comment by pedrommone** [comment]
When I suggested to respect the (semver), I mean:

![Changes](http://i.imgur.com/1wiZb7t.png)

- `laravel/framework` can be only updated to version `5.2.31`
- `phpunit/phpunit` cannot be touched

**Comment by iwex** [comment]
@pedrommone yeah, but if you want, you can also mark `major` and cross your fingers :D
